cms/djangoapps/contentstore/views/course.py

```diff
@@ -1090,8 +1098,18 @@ def advanced_settings_handler(request, course_key_string):
                     )
 
                     if is_valid:
-                        # update the course tabs if required by any setting changes
-                        _refresh_course_tabs(request, course_module)
+                        try:
+                            # update the course tabs if required by any setting changes
+                            _refresh_course_tabs(request, course_module)
+                        except InvalidTabsException as err:
+                            log.exception(err.message)
+                            response_message = [
+                                {
+                                    'message': _('An error occurred while trying to save your tabs'),
```

Comment:
So the problem is that a more specific message would probably be confusing here, because they aren't going to be editing the tabs directly from this page (the advanced settings page). It makes more sense to have the specific message in the logs, which is why we're logging the exception and putting up a generic message to the user.